openshift/system/Dockerfile

```diff
@@ -112,8 +112,9 @@ RUN yum install -y mariadb-server \
 
 FROM porta-base AS licenses-report
 RUN source /opt/app-root/etc/scl_enable \
-    && gem install license_finder \
-    && license_finder report --format=xml --save=doc/licenses/licenses.xml --decisions-file=doc/dependency_decisions.yml \
+    && sed -i -e 's/:licenses//' .bundle/config \
+    && bundle install \
```

Comment:
Why installing the whole bundle again?

Reply:
it doesn't install the whoe bundle, just adds the `licenses` group to the install